ui/src/main/webapp/src/app/components/project-list/project-list.component.ts

```diff
@@ -56,12 +54,6 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     ngOnInit():any {
         this.updateSort();
         this.getMigrationProjects();
-        this._refreshIntervalID = setInterval(() => this.getMigrationProjects(), 30000);
```

Comment:
@OndraZizka - I agree, although, I do think that it is nice to discuss them. Sometimes changes like this can have side effects as well (although probably not in this case).

But, again, no big deal. As far as I am concerned, this is settled and can go in. :)